green_cli/btc.py

```diff
@@ -309,7 +321,10 @@ def getunspentoutputs(session, details):
 @print_result
 @with_gdk_resolve
 def setunspentoutputsstatus(session, details):
-    """Set unspent outputs status. Status format is <txid>:<vout>:[default|frozen]"""
+    """Set unspent outputs status.
+
+    Status format is <txid>:<vout>:[default|frozen].
+    """
```

Comment:
Not sure about the trailing period when it's a format/command like this (ie. where punctuation characters may be part of the format/literal).

Reply:
Yes, good point, removed